pagopa/io-eudiw-app
# chore: [SIW-1241] add packages required for IT wallet integration

> [!WARNING]  
> This PR depends on #9

## Short description
This PR adds several packages needed to properly implement the IT wallet feature.

**Note:** All the code was copied directly from the `io-app` codebase, with minimal modifications.

## List of changes proposed in this pull request
- Added several packages, including
    - `@pagopa/io-react-native-cie-pid`
    - `@pagopa/io-react-native-proximity`
    - `react-native-ble-manager`
    - `react-native-webview`
    - `rn-qr-generator`
- Added some generic components used in IT wallet

## How to test
Build and run the app. You should see a simple screen with check marks on those libraries that are properly imported.
